packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -529,7 +528,8 @@ class Tabs extends Component<TabsProps, TabsState> {
     const overflowingTabProps = filteredChildrenOverflowing.map((child: React.ReactElement<TabProps>) => child.props);
 
     const uniqueId = id || getUniqueId();
-    const Component: any = component === TabsComponent.nav ? 'nav' : 'div';
+    const defaultComponent = isNav && !component ? 'nav' : 'div';
+    const Component: any = component === TabsComponent.nav || (isNav && component !== 'div') ? 'nav' : defaultComponent;
```

Comment:
532's logic is a little confusing to me, would this be equivalent and slightly cleaner?
`const Component: any = component !== undefined ? component : defaultComponent;`

Because `defaultComponent` is already set to "nav" for isNav and "div" otherwise, and if TabsComponent.nav is passed in that's should already be setting it to "nav".